tasks/go_deps.py

```diff
@@ -148,148 +119,3 @@ def send_count_metrics(
-
-                    mismatch_binaries.add(
-                        MisMatchBinary(binary, goos, goarch, new_dependencies_lines - recorded_dependencies_lines)
-                    )
-
-    if len(mismatch_binaries) > 0:
-        message = io.StringIO()
-
-        for mismatch_binary in mismatch_binaries:
-            if mismatch_binary.differences > 0:
-                message.write(
-                    color_message(
-                        f"You added some dependencies to {mismatch_binary.binary} ({mismatch_binary.os}/{mismatch_binary.arch}). Adding new dependencies to the binary increases its size. Do we really need to add this dependency?\n",
-                        "red",
-                    )
-                )
-            else:
-                message.write(
-                    color_message(
-                        f"You removed some dependencies from {mismatch_binary.binary} ({mismatch_binary.os}/{mismatch_binary.arch}). Congratulations!\n",
-                        "green",
-                    )
-                )
-
-        message.write(
-            color_message(
-                "To fix this check, please run `inv -e go-deps.generate`",
-                "orange",
-            )
-        )
-
-        raise Exit(
-            code=1,
-            message=message.getvalue(),
-        )
-
-
-@task
-def generate(
-    ctx: Context,
-):
-    for binary in BINARY_TO_TEST:
-        binary_info = BINARIES[binary]
-        entrypoint = binary_info["entrypoint"]
-        platforms = binary_info["platforms"]
-        flavor = binary_info.get("flavor", AgentFlavor.base)
-        build = binary_info.get("build", binary)
-
-        with ctx.cd(entrypoint):
-            for platform in platforms:
-                platform, arch = platform.split("/")
-
-                goos, goarch = GOOS_MAPPING[platform], GOARCH_MAPPING[arch]
-
-                filename = os.path.join(ctx.cwd, f"dependencies_{goos}_{goarch}.txt")
-
-                list = compute_binary_dependencies_list(ctx, build, flavor, platform, arch)
-
-                f = open(filename, "w")
-                f.write('\n'.join(list))
-                f.close()
-
-
-def key_for_value(map: dict[str, str], value: str) -> str:
-    """Return the key from a value in a dictionary."""
-    for k, v in map.items():
-        if v == value:
-            return k
-    raise ValueError(f"Unknown value {value}")
-
-
-@task(
-    help={
-        'build': f'The agent build to use, one of {", ".join(BINARIES.keys())}',
-        'flavor': f'The agent flavor to use, one of {", ".join(AgentFlavor.__members__.keys())}. Defaults to base',
-        'os': f'The OS to use, one of {", ".join(GOOS_MAPPING.keys())}. Defaults to host platform',
-        'arch': f'The architecture to use, one of {", ".join(GOARCH_MAPPING.keys())}. Defaults to host architecture',
-    }
-)
-def show(ctx: Context, build: str, flavor: str = AgentFlavor.base.name, os: str | None = None, arch: str | None = None):
```

Comment:
Can you leave this invoke task please ? I think it's still useful

Reply:
Agree with that. I would just remove the CI check and leave all the invoke task as they might be useful for other uscases in the future 😄